packages/webapp/pages/posts/[id]/seo.ts

```diff
@@ -0,0 +1,65 @@
+import { Post, PostType } from '@dailydotdev/shared/src/graphql/posts';
+
+export const getSeoDescription = (post: Post): string => {
+  if (post?.summary) {
+    return post?.summary;
+  }
+  if (post?.description) {
+    return post?.description;
+  }
+  return `Join us to the discussion about "${post?.title}" on daily.dev ✌️`;
+};
+
+export const getSEOJsonLd = (post: Post): string => {
+  return JSON.stringify({
+    '@context': 'https://schema.org',
+    '@type': 'Article',
+    headline: post?.title,
```

Comment:
From all of these I think this one is the only one we need to sanitize because it comes from user.

Reply:
I was thinking about someone adding `</script><script>alert("xss");</script>` but looks like `element.innerHtml` can not be attacked by that since the content is always a child node of `element.`